zkp-component/tests/zkp_requests.rs

```diff
@@ -86,6 +86,7 @@ async fn peers_dont_reply_with_outdated_proof() {
 }
 
 #[test(tokio::test)]
+#[ignore]
```

Comment:
Why are we ignoring this test now?

Reply:
Good point, we will need to regenerate the proof and add it before merging.
I added it to the PR todo list, but I will mark it as resolved so reviewing can go on.